packages/eslint-config-custom/index.js

```diff
@@ -1,10 +1,13 @@
 module.exports = {
-  extends: ['next', 'turbo', 'prettier'],
+  extends: ['next', 'turbo', 'prettier', "@feature-sliced"],
   plugins: ['@tanstack/query'],
   rules: {
     '@next/next/no-img-element': 'off',
     '@next/next/no-html-link-for-pages': 'off',
     'react/jsx-key': 'off',
+    'import/no-internal-modules': 'warn',
```

Comment:
I think this rule should be made in `info`, since it swears on all our current imports and so that the list of imports doesn't look like one solid warning. Or, can it be configured to analyze only FSD folders?

Reply:
I've made them info now. I wanna override this later for the FSD folders (maybe in #381?)

Reply:
https://github.com/curvefi/curve-frontend/pull/380/commits/ae86f99105dae540b4496b86f4df8092199d48b7